sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_recognize_linked_entities_async.py

```diff
@@ -54,50 +54,14 @@ class RecognizeLinkedEntitiesSampleAsync(object):
                 print("Url: {}".format(entity.url))
                 print("Data Source: {}".format(entity.data_source))
                 for match in entity.matches:
-                    print("Score: {0:.3f}".format(match.score))
-                    print("Offset: {}".format(match.grapheme_offset))
-                    print("Length: {}\n".format(match.grapheme_length))
+                    print("Score: {}".format(match.score))
```

Comment:
can you print `match.text` here too? and for the other linked entities sample.
print("Entity as appears in request: {}".format(match.text))